### DISCUSSION TITLE
Working together on the same project?

### DISCUSSION THREAD

**Discussion by agroenman** [asreview/asreview#1227] [Q&A]
Hi! 

Is there a possibility to work together on the same project? For example by moving the ASreview folder to a cloud location? I am noticing that it is taking a bit more time than we were anticipating, and I am hoping to help out the screening process. 

Thank you,

Annabeth

**Comment by J535D165** [comment]
Not out of the box, but we are on it. This discussion is useful to get an overview of collaboration in ASReview https://github.com/asreview/asreview/discussions/975.

And yes, you can change the location of the asreview folder to a cloud location (https://github.com/asreview/asreview/discussions/998), but this can/will cause unintentional behavior.

**Comment by Rensvandeschoot** [comment]
You can also ask this person to [export ](https://asreview.readthedocs.io/en/latest/manage.html#export-project)the project. You can import the project and continue with screening. See also this [blog ](https://asreview.nl/blog/whats-new-in-release-0-19/)post.
